lib/slack/state.ex

```diff
@@ -53,6 +53,17 @@ defmodule Slack.State do
     update_in(slack, [:groups, channel, :members], &Enum.uniq([user | &1]))
   end
 
+  def update(
+        %{type: "message", subtype: "group_topic", channel: channel, user: user, topic: topic},
+        slack
+      ) do
+    update_in(
```

Comment:
It looks like setting group topic is a an overriding update to a value according to `https://api.slack.com/events/message/group_topic`. If that's the case, could we use `put_in`?